src/main/java/core/basesyntax/model/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.model;
+
+import static java.lang.System.lineSeparator;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int height;
+    private int longSide;
+    private int shortSide;
+
+    public IsoscelesTrapezoid(String color, int height, int longSide, int shortSide) {
+        super(color);
+        this.height = height;
+        this.longSide = longSide;
+        this.shortSide = shortSide;
+    }
+
+    @Override
+    public double getArea() {
+        return ((height + longSide) / 2.0) * height;
```

Comment:
division result of integers is not what you expect (5/2=2 in your case)

Reply:
I edited it but I don't understand why? I'm using 2.0 to convert the result to a double